src/tests/integration/test-data/foundry/test/BlockParamsTest.t.sol

```diff
@@ -36,3 +36,15 @@ contract BlockParamsTest is Test {
         assert(x >= 0);
     }
 }
+
+contract RollTest is Test {
+    function setUp() external {
+        vm.roll(123);
+    }
+
+    // function testFail_run1() external {
```

Comment:
Sorry, that came from the contract I was running — we probably don't need it here :)

Reply:
Done. I need to update the expected output once more.